src/app/visual/responsive/responsive-demo.component.html

```diff
@@ -0,0 +1,52 @@
+<div id="screenshot-responsive">
+  <div
+    id="screen-queries"
+  >
+    <sky-responsive-container></sky-responsive-container>
+  </div>
+  <div
+    id="container-queries"
+    [ngClass]="{
+      'sky-media-container-xs': currentBreakpoint === 'xs',
+      'sky-media-container-sm': currentBreakpoint === 'sm',
+      'sky-media-container-md': currentBreakpoint === 'md',
+      'sky-media-container-lg': currentBreakpoint === 'lg'
+    }"
+  >
+    <sky-responsive-container></sky-responsive-container>
+  </div>
+</div>
+<div>
+  <button
+    class="sky-btn sky-btn-primary"
+    id="xs-toggle"
+    type="button"
+    (click)="setClass('xs')"
+  >
+    Set xs container
+  </button>
+  <button
+    class="sky-btn sky-btn-primary"
+    id="sm-toggle"
+    type="button"
+    (click)="setClass('sm')"
+  >
+    Set sm container
+  </button>
+  <button
+    class="sky-btn sky-btn-primary"
+    id="md-toggle"
+    type="button"
+    (click)="setClass('md')"
+  >
+    Set md container
+  </button>
+  <button
+    class="sky-btn sky-btn-primary"
+    id="lg-toggle"
+    type="button"
+    (click)="setClass('lg')"
+  >
+    Set lg container
+  </button>
+</div>
```

Comment:
It would be super awesome if this demo also showed a variable showing what breakpoint the screen is at. Also, for super bonus points, another display showing what variable the "container" is set at (so you don't have to inspect code).

Reply:
Done